cmd/brew-test-bot.rb

```diff
@@ -977,7 +980,10 @@ def test_ci_upload(tap)
 
     if (pr = ENV["UPSTREAM_PULL_REQUEST"])
       pull_pr = "https://github.com/#{tap.user}/homebrew-#{tap.repo}/pull/#{pr}"
-      safe_system "brew", "pull", "--clean", pull_pr
+      args = []
+      args << "--tap=#{@tap}" unless @tap.nil?
+      args << pull_pr
+      test "brew", "pull", "--clean", *args
```

Comment:
The Circle CI configuration runs `brew test-bot`, which creates the bottles. `brew install --build-bottle` creates the keg and `brew bottle` then creates the bottle. The `brew install` and `brew bottle` commands are inside `brew test-bot`, so the code to change the bottle tag of `INSTALL_RECEIPT.json` must live inside `brew test-bot`. I suggest using `inreplace` to change the `bottle` tag of `INSTALL_RECEIPT.json` between `brew install --bottle` and `brew bottle`. Does that work for you, or can you suggest an alternative solution?